includes/Core/Modules/Module.php

```diff
@@ -570,11 +570,12 @@ private function make_data_request( Data_Request $data ) {
 	 * @param int    $offset     Days the range should be offset by. Default 1. Used by Search Console where
 	 *                           data is delayed by two days.
 	 * @param bool   $previous   Whether to select the previous period. Default false.
+	 * @param bool   $day_align  Whether to align the previous period days of the week to current period. Default false.
 	 *
 	 * @return array List with two elements, the first with the start date and the second with the end date, both as
 	 *               'Y-m-d'.
 	 */
-	protected function parse_date_range( $range, $multiplier = 1, $offset = 1, $previous = false ) {
+	public function parse_date_range( $range, $multiplier = 1, $offset = 1, $previous = false, $day_align = false ) {
```

Comment:
Let's not change the visibility here as that only seems to be done for testing purposes. While it's a little less clean, let's use `Reflection` to make the method accessible and invoke it in the test instead. See `\Google\Site_Kit\Tests\Core\Util\Activation_NoticeTest::assertAssetsEnqueued` for an example.

Reply:
Aha, thanks for the tip!

Reply:
Adjusted